platform/chronos/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG RUST_VERSION=1.80.1
+ARG APP_NAME=discord
+
+FROM lukemathwalker/cargo-chef:latest-rust-${RUST_VERSION} AS chef
+ARG APP_NAME
+WORKDIR /app
+
+FROM chef AS planner
+COPY . .
+
+RUN cargo chef prepare  --recipe-path recipe.json
+
+FROM chef AS builder
+COPY --from=planner /app/recipe.json recipe.json
```

Comment:
I assume these files are just part of the image?

Reply:
yea, this is part of the chef builder, seems to be the recommended path to avoid pains of building docker on rust, found it in the book Zero to Production in Rust